app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt

```diff
@@ -20,8 +20,8 @@ import kotlinx.android.synthetic.main.main_fragment.*
 
 class MainFragment : Fragment() {
 
-    val CAMERA_PERMISSION_REQUEST_CODE = 1999
-    val CAMERA_REQUEST_CODE = 1998
+    private val CAMERA_PERMISSION_REQUEST_CODE = 1999
```

Comment:
visibility modifiers according to:
https://kotlinlang.org/docs/reference/coding-conventions.html